src/Umbraco.Core/Persistence/Factories/DomainFactory.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Umbraco.Core.Models;
+using Umbraco.Core.Persistence.Dtos;
+
+namespace Umbraco.Core.Persistence.Factories
+{
+    internal static class DomainFactory
```

Comment:
Moved from internal sub-class in `DomainRepository` and now aligns with other factories within this namespace.